### PR TITLE
change the redme file

### PR DESCRIPTION
**Pull request by imvisp**
-added some subpoints 
-added pictures.

### CODE REVIEW

**Review by vinzvinci — COMMENTED** [review]
Hmm can you screenshot the work you did? Coz this will help us to know the results better and faster then we can look to the code after. And I suggest rename this pull request `title` to something related to what you did. Example: `docs: added mlh picture and subtopic` - just example. 

> By the way, thank you for contributing :)

**Comment by imvisp** [comment]
> Hmm can you screenshot the work you did? Coz this will help us and the maintainer to know the results better and faster then we can look to the code after. And I suggest rename this pull request `title` to something related to what you did. Example: `docs: added mlh picture and subtopic` - just example.
> 
> > By the way, thank you for contributing :)

i made changes in pr you could see that.

**Comment by muthuannamalai12** [comment]
@imvisp Can we replace that s as Capital S in line 12

**Comment by vinzvinci** [comment]
> > Hmm can you screenshot the work you did? Coz this will help us and the maintainer to know the results better and faster then we can look to the code after. And I suggest rename this pull request `title` to something related to what you did. Example: `docs: added mlh picture and subtopic` - just example.
> > > By the way, thank you for contributing :)
> 
> i made changes in pr you could see that.

I'm just suggesting, what I mean is provide more details before we go jump from it.

**Comment by kcoder63** [comment]
Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too

**Comment by vinzvinci** [comment]
> Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too

- @muthuannamalai12 first. (discuss about lhd share and added picture). 

- Then make changes again @imvisp, `more commit` = `more green square`. You focus on steps right and added accurate picture?

**Comment by imvisp** [comment]
> > Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too
> 
> - @muthuannamalai12 first. (discuss about lhd share and added picture for the LHD). 
> 
> - Then make changes again @imvisp, `more commit` = `more green square`. You focus on steps right and added accurate picture for the steps?

Yes

File: README.md [hunk 1]
```diff
@@ -4,7 +4,13 @@
 <img align="center" src="1.png" style="width:100%">  
 <h2>2. Login </h2> 
 <h2>3. Go to the Profile Section & fill the required 
-details</h2>
+details like this</h2>
+<h3>Add your first and last name</h3>
+<img align="center" src="main1.png" style="width:100%">
+<h3>Add your Email</h3>
+<img align="center" src="main2.png" style="width:100%">
+<h3>select this</h3>
```
Comment (kcoder63) [inline README.md:12]: I'd suggest Accept Discord rules and Code of Conduct rather than select this.

**Review by kcoder63 — CHANGES_REQUESTED** [review]

**Comment by kcoder63** [comment]
> > Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too
> 
> * @muthuannamalai12 first. (discuss about lhd share and added picture for the LHD).
> * Then make changes again @imvisp, `more commit` = `more green square`. You focus on steps right and added accurate picture for the steps?

If I merge this, won't the changes @muthuannamalai12 made overridden? Since this PR is changes made on the older one

**Comment by imvisp** [comment]
> > > Which PR should I merge now? There are two with change the readme @muthuannamalai12 made one too
> > 
> > * @muthuannamalai12 first. (discuss about lhd share and added picture for the LHD).
> > * Then make changes again @imvisp, `more commit` = `more green square`. You focus on steps right and added accurate picture for the steps?
> 
> If I merge this, won't the changes @muthuannamalai12 made overridden? Since this PR is changes made on the olderAre you talking about my pr or muthu's ?

**Comment by kcoder63** [comment]
Yours, I merged muthu's PR first as suggested, now if I merged yours it might override his changes because you made changes on an older README

**Comment by imvisp** [comment]
> Yours, I merged muthu's PR first as suggested, now if I merged yours it might override his changes because you made changes on an older README

So what I have to do now ?

File: README.md (hunk 1 above)
Comment (imvisp) [reply README.md:12]: Ok will do this 👍

**Comment by kcoder63** [comment]
Um yeah that should do

**Comment by imvisp** [comment]
> Um yeah that should do

what about the old one can you delete ?

**Comment by kcoder63** [comment]
I'll close this and sorry for the trouble :(

**Pull request closed without merging**